ieee-trans-med-imaging.csl

```diff
@@ -0,0 +1,349 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>IEEE Transactions on Medical Imaging</title>
+    <id>http://www.zotero.org/styles/ieee-trans-med-imaging</id>
+    <link href="http://www.zotero.org/styles/ieee-trans-med-imaging" rel="self"/>
```

Comment:
The ID and self-link should be in full. See the current TMI style in the repository.
Travis is not happy as it found another style in the repository with the same ISSN. Obviously the auto-generated dependent style will need to be taken out after.